src/models/TraktItem.ts

```diff
@@ -1,4 +1,4 @@
-import { Moment } from 'moment';
+import * as moment from 'moment';
```

Comment:
Why this change?

Reply:
Because I'm now converting a date in the `load` method.